json-schemas/workflowRun.json

```diff
@@ -0,0 +1,29 @@
+{
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "integer"
+        },
+        "entity_id": {
+            "type": "integer"
+        },
```

Comment:
Is this JSON schema used for the graphQL response from mesh?  We'll want the IDs to be strings, even though they are integers in Rails, correct?